odoo/addons/base/tests/test_ir_filters.py

```diff
@@ -328,3 +328,15 @@ def test_filters(self):
                     order=','.join(ast.literal_eval(filter_.sort)),
                     context=context,
                 )
+
+    def test_filters_dont_have_action_on_action_delete(self):
+        res_partner_model = self.env.ref('base.model_res_partner')
+        action = self.env['ir.actions.server'].create({
+            'name': 'test_action',
+            'model_id': res_partner_model.id,
+            'state': 'object_write',
+        })
+        Filters = self.env['ir.filters']
+        filter = Filters.create(dict(name="Test Filter", model_id="ir.filters", action_id=action.id))
```

Comment:
it's a detail but usually we use the equivalent syntactic sugar instead of dict so it looks slightly odd

```suggestion
        filter = Filters.create({"name": "Test Filter", "model_id": "ir.filters", "action_id": action.id})
```